frontend/src/js/controllers/challengeCtrl.js

```diff
@@ -581,11 +581,23 @@
             parameters.callback = {
                 onSuccess: function(response) {
                     var details = response.data;
-                    vm.submissionResult = details;
 
                     for (var i = 0; i < details.results.length; i++) {
                         vm.submissionVisibility[details.results[i].id] = details.results[i].is_public;
+                        if (details['results'][i]['method_name'] == "null") {
```

Comment:
You probably need a better way of comparison. Think about the cases where the value can be `""`, `"    "`, `null`, `"null"`. Similar is the case with other fields.

Reply:
@deshraj Since the database stores the values as `null` even when the participant leaves the field as empty, also if participant fills in `null` value then also it won't break.